environment.go

```diff
@@ -468,6 +453,27 @@ func (e *envState) getRemoteQueueGroupDBOptions() queue.MongoDBOptions {
 	return opts
 }
 
+func (e *envState) getNamedRemoteQueueOptions() map[string]queue.MongoDBQueueOptions {
+	perQueueOpts := map[string]queue.MongoDBQueueOptions{}
+	for _, namedQueue := range e.settings.Amboy.NamedQueues {
+		dbOpts := e.getRemoteQueueGroupDBOptions()
+		if namedQueue.SampleSize != 0 {
```

Comment:
It seems to me that if namedQueue.SampleSize is zero, dbOpts will default to 0. Why not just set this to namedQueue.SampleSize?

Reply:
Mostly just to leave open the future possibility that remote queue group default fields such as sample size can be set and respected. The options for named queues should only override the defaults if they're explicitly set and otherwise should obey whatever default is set. Doing this zero/non-zero check when following options precedence/defaulting rules is a common thing in our code bases.